spec/services/caution_decision_tree_spec.rb

```diff
@@ -4,68 +4,54 @@
   let(:disclosure_check) do
     instance_double(
       DisclosureCheck,
-      caution_type: caution_type,
-      condition_complied: condition_complied,
-      is_date_known: is_date_known,
```

Comment:
if `is_date_known`  not used can we please remove it in another PR.

Reply:
Yes. I would do in a separate PR.
It has not been used for a while now.